tdp/core/models/deployment_log.py

```diff
@@ -230,7 +292,7 @@ def from_stale_components(
         Raises:
             NothingToReconfigureError: If no component needs to be reconfigured.
         """
-        operation_host_names: set[tuple[str, str]] = set()
+        operation_host_names: set[Tuple[str, str]] = set()
```

Comment:
You should rebase on master to take [#431](https://github.com/TOSIT-IO/tdp-lib/pull/431) into account (this line shouldn't be modified).

Reply:
Done